hls4ml/model/profiling.py

```diff
@@ -589,6 +589,7 @@ def get_ymodel_keras(keras_model, X):
         name = layer.name
         if (
             hasattr(layer, 'activation')
+            and hasattr(layer.activation, "__name__")
```

Comment:
Is this really needed? I think there were two parallel fixes for this in two pull requests, and all layer.acitvations that are of type `Activation` or `QActivation` should have __name__ defined, right?